airflow/www/templates/airflow/trigger.html

```diff
@@ -44,12 +44,23 @@ <h2>Trigger DAG: {{ dag_id }}</h2>
     <div class="form-group row">
       <div class="col-md-2">
         <label for="run_id">Run id (Optional)</label>
-        <input type="text" class="form-control" placeholder="Run ID" name="run_id" >
+        <input type="text" class="form-control" placeholder="Run ID" name="run_id">
+      </div>
+    </div>
+    <div class="form-group row">
+      <div class="col-md-2">
+        <label for="recent_configs">Recent Configurations</label>
+        <select class="form-control" name="recent_configs" id="recent_configs">
+          <option value="">None</option>
```

Comment:
Maybe we can display something like 'empty config' instead of None that doesn't provide much information to the user ?